tests/python/test-parameter-server.py

```diff
@@ -0,0 +1,368 @@
+    <collision>\
+      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\
+      <geometry>\
+        <box size=\"0.1 0.1 0.1\"/>\
+      </geometry>\
+    </collision>\
+    <visual>\
+      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\
+      <geometry>\
+        <mesh filename=\"package://rrbot_description/meshes/hokuyo.dae\"/>\
+      </geometry>\
+    </visual>\
+    <inertial>\
+      <mass value=\"1e-5\"/>\
+      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\
+      <inertia ixx=\"1e-6\" ixy=\"0\" ixz=\"0\" iyy=\"1e-6\" iyz=\"0\" izz=\"1e-6\"/>\
+    </inertial>\
+  </link>\
+  <joint name=\"camera_joint\" type=\"fixed\">\
+    <axis xyz=\"0 1 0\"/>\
+    <origin rpy=\"0 0 0\" xyz=\"0.05 0 0.9\"/>\
+    <parent link=\"link3\"/>\
+    <child link=\"camera_link\"/>\
+  </joint>\
+  <!-- Camera -->\
+  <link name=\"camera_link\">\
+    <collision>\
+      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\
+      <geometry>\
+        <box size=\"0.05 0.05 0.05\"/>\
+      </geometry>\
+    </collision>\
+    <visual>\
+      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\
+      <geometry>\
+        <box size=\"0.05 0.05 0.05\"/>\
+      </geometry>\
+      <material name=\"red\"/>\
+    </visual>\
+    <inertial>\
+      <mass value=\"1e-5\"/>\
+      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\
+      <inertia ixx=\"1e-6\" ixy=\"0\" ixz=\"0\" iyy=\"1e-6\" iyz=\"0\" izz=\"1e-6\"/>\
+    </inertial>\
+  </link>\
+  <!-- generate an optical frame http://www.ros.org/reps/rep-0103.html#suffix-frames\
+      so that ros and opencv can operate on the camera frame correctly -->\
+  <joint name=\"camera_optical_joint\" type=\"fixed\">\
+    <!-- these values have to be these values otherwise the gazebo camera image\
+        won't be aligned properly with the frame it is supposedly originating from -->\
+    <origin rpy=\"-1.57079632679 0 -1.57079632679\" xyz=\"0 0 0\"/>\
+    <parent link=\"camera_link\"/>\
+    <child link=\"camera_link_optical\"/>\
+  </joint>\
+  <link name=\"camera_link_optical\">\
+  </link>\
+  <transmission name=\"tran1\">\
+    <type>transmission_interface/SimpleTransmission</type>\
+    <joint name=\"joint1\">\
+      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>\
+    </joint>\
+    <actuator name=\"motor1\">\
+      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>\
+      <mechanicalReduction>1</mechanicalReduction>\
+    </actuator>\
+  </transmission>\
+  <transmission name=\"tran2\">\
+    <type>transmission_interface/SimpleTransmission</type>\
+    <joint name=\"joint2\">\
+      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>\
+    </joint>\
+    <actuator name=\"motor2\">\
+      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>\
+      <mechanicalReduction>1</mechanicalReduction>\
+    </actuator>\
+  </transmission>\
+</robot>\
+"
+
+# 
+model = pin.buildModelFromXML(urdf_rrbot_model_string)
+
+print("model.nq:"+str(model.nq))
```

Comment:
```python
print("model.nq", model.nq)
```

would be a bit more idiomatic.

(or even `print(f"{model.nq=}")`, in python >= 3.8 )